### PR TITLE
Upgrade SonarQube to 6.7.4 & PostgreSQL to 10.4

### PR DESCRIPTION
**Pull request by BalmungSan**
Hope everything is ok.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,34 +23,36 @@ docker run -d --name sonarqube-scala-plugins-full \
...
-[2.1.0](https://github.com/mwz/sonarqube-scala-docker/releases/tag/2.1.0), [2.0.0](https://github.com/mwz/sonarqube-scala-docker/releases/tag/2.0.0) | 6.7.1 LTS ([documentation](https://docs.sonarqube.org/display/SONARQUBE67/Documentation)) ([changelog](https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=10930&version=14137)) | [6.0.0](https://github.com/mwz/sonar-scala/releases/tag/v6.0.0) | [1.3.0](https://github.com/arthepsy/sonar-scala-extra/releases/tag/v1.3.0)
-[1.0.0](https://github.com/mwz/sonarqube-scala-docker/releases/tag/1.0.0) | 5.6.7 LTS ([documentation](https://docs.sonarqube.org/display/SONARQUBE56/Documentation)) | [0.0.3](https://github.com/Sagacify/sonar-scala/releases/tag/v0.0.3) | [1.3.0](https://github.com/arthepsy/sonar-scala-extra/releases/tag/v1.3.0)
+Version | SonarQube | PostgreSQL | sonar-scala | sonar-scala-extra
```
Comment (mwz) [inline README.md]: The reason I didn't include the postgres column is that I haven't been versioning the docker-compose.yml file - just the docker images themselves, which don't have any dependency on postgres. Tbh, I don't think it's worth tracking the postgres version here.

File: README.md [hunk 2]
```diff
@@ -61,4 +63,4 @@ Version | SonarQube | sonar-scala | sonar-scala-extra
 - **1.0.0** - SonarQube 5.6.7 LTS.
 
 ## License
-The project is licensed under the Apache License v2. See the [LICENSE file](LICENSE) for more details.
+The project is licensed under the Apache License v2. See the [LICENSE file](LICENSE) for more details.|
```
Comment (mwz) [inline README.md]: Could you please remove the `|`?

File: README_DOCKERHUB.md [hunk 3]
```diff
@@ -4,35 +4,37 @@ Docker images with out-of-the-box [SonarQube LTS](https://www.sonarqube.org) ins
 
 There are two types of images available: images with [sonar-scala](https://github.com/mwz/sonar-scala) and [sonar-scala-extra](https://github.com/arthepsy/sonar-scala-extra) plugins, which can be mounted as a volume into a SonarQube container and images which bundle those plugins with SonarQube (suffixed with `-full`). 
 
-- `2.5.0`, `latest` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.5.0/Dockerfile), [(v2.5.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.5.0)
+- `2.6.0`, `latest` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.6.0/Dockerfile), [(v2.6.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.6.0)
```
Comment (mwz) [inline README_DOCKERHUB.md]: Looks like you missed the `2.6.0-full` version here.

File: README_DOCKERHUB.md [hunk 4]
```diff
@@ -4,35 +4,37 @@ Docker images with out-of-the-box [SonarQube LTS](https://www.sonarqube.org) ins
...
-2.3.0 | | 6.2.0 | 1.3.0
-2.3.0-full | 6.7.3 LTS | 6.2.0 | 1.3.0
-2.2.1 | | 6.1.0 | 1.3.0
-2.2.1-full | 6.7.3 LTS | 6.1.0 | 1.3.0
-2.2.0 | | 6.1.0 | 1.3.0
-2.2.0-full | 6.7.2 LTS | 6.1.0 | 1.3.0
-2.1.0 | | 6.0.0 | 1.3.0
-2.1.0-full | 6.7.1 LTS | 6.0.0 | 1.3.0
-
+Version | SonarQube | PostgreSQL | sonar-scala | sonar-scala-extra
```
Comment (mwz) [inline README_DOCKERHUB.md]: As mentioned previously, Postgres isn't included in the docker images and we should exclude the `PostgreSQL` column from this table.

**Review by mwz — CHANGES_REQUESTED** [review]

**Comment by BalmungSan** [comment]
Roger, I just pushed the changes 😄

**Review by mwz — APPROVED** [review]
Thanks! It looks good, I'll merge it and publish the images later today.

**Pull request merged**